redun/executors/aws_batch.py

```diff
@@ -309,6 +322,13 @@ def run_docker(
     for host, container in volumes:
         common_args.extend(["-v", f"{host}:{container}"])
 
+    common_args.extend([f"--memory={memory}g", f"--cpus={vcpus}"])
+    if gpus:
+        # We can't easily assign a single gpu so we just make all available if any GPUs are required
```

Comment:
This line is one character too long and is triggering this issues in the lint build:

```
redun/executors/aws_batch.py:327:100: E501 line too long (100 > 99 characters)
```

Consider changing to

```suggestion
        # We can't easily assign a single gpu so we make all available if any GPUs are required.
```

Reply:
Done